scripts/create-agent.ts

```diff
@@ -577,6 +577,19 @@ AGENT_ENVIRONMENT=production`;
           tag: "PROFILE",
           command: "[ -f ~/.profile ] && . ~/.profile || true",
         },
+        {
+          tag: "NODE_MODULES",
+          command: "cd /home/ubuntu && npm install playwright",
```

Comment:
Remove all three commands for now. We're okay with playwright not working for the time being

Reply:
Remove the commands. If CI fails as a result, let me see what the ci failure is